core/controllers/admin.py

```diff
@@ -274,7 +282,7 @@ def _generate_dummy_explorations(
                                'Elvish, language of "Lord of the Rings',
                                'The Science of Superheroes']
             exploration_ids_to_publish = []
-            for i in range(num_dummy_exps_to_generate):
+            for i in python_utils.RANGE(num_dummy_exps_to_generate):
```

Comment:
What is the rationale behind this naming convention (as opposed to python_utils.range)?

Reply:
I am defining RANGE as a class level constant and that's why it needs to be in caps. Also I don't think we should redefine a built-in python function name